server/models/User.py

```diff
@@ -12,8 +12,7 @@ def load_user(user_id):
 
 
 class User(db.Model, UserMixin):
-    id = db.Column(db.Integer, primary_key=True)
-    public_id = db.Column(db.String(50), unique=True)
+    id = db.Column(db.String(50), primary_key=True, unique=True)
```

Comment:
In this case, `unique` is redundant (`primary_key` already forces the key to be unique).

Reply:
Done!